components/camel-spring-redis/src/main/java/org/apache/camel/component/redis/processor/idempotent/SpringRedisIdempotentRepository.java

```diff
@@ -95,7 +107,12 @@ public boolean confirm(String key) {
 
     @Override
     protected void doStart() throws Exception {
-        // noop
+        if (redisConfiguration == null) {
```

Comment:
hello @davsclaus ,
could you help me understand how this redisConfiguration should be initialized here? 
Because in my case it is just null which basically initializes redisTemplate to default values. 
What i am doing is:
```
    @Bean
    public SpringRedisStringIdempotentRepository idempotentRepository(RedisTemplate<String, String> redisTemplate) {
        SpringRedisStringIdempotentRepository idempotentRepository = new SpringRedisStringIdempotentRepository(redisTemplate, IDEMPOTENT_REPOSITORY_KEY);
        idempotentRepository.setExpiry(10000L);
        return idempotentRepository;
    }
```
Feedback appreciated!